test/cpo/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 file(GLOB HEADERS "*.h")
 file(GLOB SOURCES "*.cpp")
 
-add_executable (cpo_test ${HEADERS} ${SOURCES})
+add_executable (cpo_test ${HEADERS} CPOTest.cpp Main.cpp)
+add_executable(cpo_client_test CPOClientTest.cpp)
 
 target_link_libraries (cpo_test PRIVATE appbase Seastar::seastar dto tso_client)
+target_link_libraries (cpo_client_test PRIVATE appbase Seastar::seastar dto tso_client cpo_client transport common)
```

Comment:
missing new line at end of file. You should auto-enable this in your editor settings

Reply:
This is weird. I've had it turned on in my editor and have checked manually that there is an ending new line in my local file.